DQM/SiPixelPhase1Track/plugins/SiPixelPhase1TrackClusters.cc

```diff
@@ -99,10 +114,26 @@ namespace {
     pixelClusterShapeCacheToken_ =
         consumes<SiPixelClusterShapeCache>(iConfig.getParameter<edm::InputTag>("clusterShapeCache"));
 
-    trackerTopoToken_ = esConsumes<TrackerTopology, TrackerTopologyRcd>();
-    trackerGeomToken_ = esConsumes<TrackerGeometry, TrackerDigiGeometryRecord>();
+    trackerTopoToken_ = esConsumes<edm::Transition::BeginRun>();
+    trackerGeomToken_ = esConsumes<edm::Transition::BeginRun>();
     clusterShapeHitFilterToken_ =
         esConsumes<ClusterShapeHitFilter, CkfComponentsRecord>(edm::ESInputTag("", "ClusterShapeHitFilter"));
+    templateDBobjectToken_ = esConsumes<edm::Transition::BeginRun>();
+  }
+
+  void SiPixelPhase1TrackClusters::dqmBeginRun(const edm::Run& iRun, const edm::EventSetup& iSetup) {
+    // get geometry
+    edm::ESHandle<TrackerGeometry> tracker = iSetup.getHandle(trackerGeomToken_);
+    assert(tracker.isValid());
```

Comment:
How about just
```cpp
const TrackerGeometry& tracker = iSetup.get(trackerGeomToken_);
```
? Or actually, `tracker` seems to be unused so how about removing these two lines?